experimental/MatroidRealizationSpaces/src/realization_space.jl

```diff
@@ -33,25 +33,25 @@ end
 
 function Base.show(io::IO, ::MIME"text/plain", RS::MatroidRealizationSpace)
   if has_attribute(RS, :is_realizable) && !is_realizable(RS)
-    if RS.char == nothing && RS.q == nothing
+    if RS.char === nothing && RS.q === nothing
       print(io, "The matroid is not realizable.")
     else
       print(io, "The matroid is not realizable over the specified field or characteristic.")
     end
   else
+    io = pretty(io)
     if RS.one_realization
       println(io, "One realization is given by")
     elseif has_attribute(RS, :is_realizable) && is_realizable(RS)
       println(io, "The realizations are parametrized by")
     elseif !has_attribute(RS, :is_realizable)
       println(io, "The realization space is")
     end
-
+    print(io, Indent())
     show(io, MIME("text/plain"), RS.realization_matrix)
-    print(io, "\nin the ", RS.ambient_ring)
+    print(io, "\n", Dedent(), "in the ", Lowercase(), RS.ambient_ring)
     I = RS.defining_ideal
-    if (I isa NumFieldOrdIdl && I.gen != ZZ(0)) ||
-      (I isa Ideal && !iszero(I))
+    if !iszero(I)
       print(io, "\nwithin the vanishing set of the ideal\n", I)
```

Comment:
probably should have an `Indent` here, too, and also in the final `print` after the fold -- but of course that's beyond the scope of this PR, just wanted to mention it.

Reply:
Yeah, I played around with some things here but wasn't happy with all of them. So that got postponed